src/app/shared/directives/highlight.directive.ts

```diff
@@ -75,9 +70,7 @@ export class HighlightDirective {
 
   private highlight(): void {
     const element: HTMLElement = this.elementRef.nativeElement;
-    const schema: HighlightSchemas = this.getCourseSchema(
-      this.creationDate,
-    );
+    const schema: HighlightSchemas = this.getCourseSchema(this.creationDate);
 
     for (const key in schema) {
```

Comment:
keys(schema).forEach(key => { if (key) { this.renderer.addClass(element, this.schemas[key]) } } )

and we have to talk about renderer